### DISCUSSION TITLE
All my repos, including forks are included in Codespaces quota?

### DISCUSSION THREAD

**Discussion by wintercounter** [community/community#40101] [Codespaces] [answered]
I've just got an e-mail warning me that I've used 75% of my Codespaces storage. This is odd, because I never actually used Codespaces. The only explanation comes in my mind that all my repositories, including forks are included for whatever reason.

**Comment by louh** [comment, marked as answer]
I just ran into this issue, and based on this conversation [in this thread](https://github.com/orgs/community/discussions/39133), it seems this recently rolled out and it's buggy as hell. For one thing, old Codespaces are calculated as accumulating storage every single day, even if you just have one of them and never use it. So one old Codespace at 30 days a month is like saying you have 30 repos using Codespaces a month. It chews through the usage limit _incredibly_ quickly.

Here's what to do:
- See your list of Codespaces here: https://github.com/codespaces (they really need to link this somewhere that's easier to get to)
- Delete all Codespaces (probably one or two you tried, years ago)
- In a few days, double check the [usage report](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/viewing-your-github-codespaces-usage) and make sure it has stopped accruing usage. The last date should be the day you deleted the codespaces.

**Reply by wintercounter** [reply]
Seems like I clicked a Codespace button before for a 3rd party repo and it was just sitting there. Still, it's just wrong how this works :D

**Comment by Hartorn** [comment]
I'm re-using this, I also got notifications, but I have nothing into https://github.com/codespaces
Anyone knows who to contact in this case ?
